ONSdigital/sde-bdd
# Update sde tests

**Changes**

- Migrate to using new eq key names
- Use onsdigital flask image
- Correct tests to count files from ftp subfolder structure

**How to test**

- Spin up a docker-compose app from the dockers repo adding the following service to the .yml file:

  ```
bdd:
    build: ./sde-bdd
    ports:
      - "5050:5000"
    links:
      - posie:posie
      - rabbit:rabbit
      - pure-ftpd:pure-ftpd
    volumes:
      - ./sde-bdd:/app
      - ./jwt-test-keys:/keys
    env_file:
      - rabbit.env
      - ftp.env
```

- Run the performance tests via the exposed server ip that docker starts with on port 5050
- Confirm performance tests at the endpoint '/publish/performance.feature' can be published and pass (will appear to hang for 60s as runs inline)

**Who can test**

Anyone but @iwootten
